config/prod.config.exs

```diff
@@ -0,0 +1,13 @@
+import Config
+
+config :roule_avec_spot, Youtube,
+  client_secret: System.get_env("YOUTUBE_OAUTH_CLIENT_SECRET"),
```

Comment:
You might want to use the [`fetch_env!`](https://hexdocs.pm/elixir/System.html#fetch_env!/1) bang version here to fail early and detect missing configs early (vs propagating a `nil` causing an error downstream)